target/linux/generic/files/drivers/net/phy/rtl8261n/phy_patch.c

```diff
@@ -170,10 +175,14 @@ int32 phy_patch(uint32 unit, rtk_port_t port, uint8 portOffset, uint8 patch_mode
             break;
         }
     }
+    ret = _phy_patch_process(unit, port, portOffset, rtl826XB_patch_rtk_conf, sizeof(rtl826XB_patch_rtk_conf), patch_mode);
```

Comment:
this patch is realtek only, but was pushed to the generic tree. i dont know if this will harm the mediatek tagets. please make this exclusive for realtek. i think this code isnt used for realtek the patches are applied for anything else